.gitignore

```diff
@@ -40,3 +40,5 @@ docs/_build
 # PyCharm
 .idea
 .idea/*
+
+gaff_bugs.txt
```

Comment:
It looks like you added your local .gitignore to one of the commits. Can you remove these two lines you added and re-commit?

Reply:
That's strange, that change was on a different branch and I specifically didn't add/commit it. I'll go ahead and fix that now